config-v3-documentation

```diff
@@ -55,6 +55,20 @@ data_subdir=tests/
 in_gen=src/gen
 # Source code of generator (without suffix)
 
+in_format=text
+# Format of input:
+#   text    text/plain in ASCII (default)
+#           subject to normalization (newlines, encoding)
+#   binary  application/binary
```

Comment:
This is not a real MIME type. Did you mean `application/octet-stream`?

Reply:
Right. But I would much prefer having something imediately clear...

Reply:
If it's not supposed to be a MIME type, then I don't understand what you mean by "application/binary". Do you mean the input/output will be either an application or a sequence of bits?